extractor/src/extract/extractor.py

```diff
@@ -13,17 +13,17 @@
 from arkhn_monitoring import Timer
 
 
-SQL_RELATIONS_TO_METHOD = {
-    "<": "__lt__",
-    "<=": "__le__",
-    "<>": "__ne__",
-    "=": "__eq__",
-    ">": "__gt__",
-    ">=": "__ge__",
+SQL_RELATIONS_TO_METHOD: Dict[str, Callable[[AlchemyColumn, str], Callable]] = {
+    "<": lambda col, value: col.__lt__(value),
```

Comment:
I think you can write `col < value` here, right?

Reply:
I will keep the method as a function for consistency with pagai